vms/avm/network/network.go

```diff
@@ -116,71 +204,59 @@ func (n *network) AppGossip(ctx context.Context, nodeID ids.NodeID, msgBytes []b
 		return nil
 	}
 
-	if err := n.issueTx(tx); err == nil {
+	err = n.mempool.Add(tx)
+	if err == nil {
 		txID := tx.ID()
+		n.txPushGossiper.Add(tx)
```

Comment:
Can you confirm that we will now gossip any txs that are added to the mempool (even if they were recently gossiped)?

AFAICT this is true and am just calling out because it is a behavior change.

Reply:
Previously we would:
> Push gossip a tx that is in our mempool whenever we heard about it unless it was recently gossiped

Now we will:
> Push gossip a tx after adding it to our mempool when hearing about it

It is slightly different, but I wouldn't really expect it to be too much of a difference (unless we are frequently rejecting blocks... Because previously we would have still dropped recent txs, but we could add a tx a second time after building a block if that block becomes no longer preferred)